bliss/encoder.py

```diff
@@ -87,12 +90,15 @@ def max_a_post(self, image_ptiles: Tensor) -> Dict[str, Tensor]:
             - 'galaxy_bools', 'star_bools', and 'galaxy_probs' from BinaryEncoder.
             - 'galaxy_params' from GalaxyEncoder.
         """
-        var_params = self.location_encoder.encode(image_ptiles)
+        log_image = subtract_bg_and_log_transform(image, background)
```

Comment:
The transforms used for each model are the same as before. The only difference is that the LocationEncoder and BinaryEncoder now use the background instead of "image_ptiles.min()". 

I found that removing the log transform on the LocationEncoder led to a significant drop in performance on the M2 GPU test. I haven't run a comparison of different transformations on the VAE.